docs/sources/developers/plugins/metadata.md

```diff
@@ -15,60 +15,69 @@ The plugin.json file is required for all plugins. When Grafana starts, it scans
 | `hiddenQueries` | boolean                 | No       |                                                                                                                                                                                                                                                                                                                                                                                                         |
-| `includes`      | [object](#includes)[]   | No       |                                                                                                                                                                                                                                                                                                                                                                                                         |
+| `includes`      | [object](#includes)[]   | No       | Resources to include in plugin.                                                                                                                                                                                                                                                                                                                                                                         |
 | `logs`          | boolean                 | No       | For data source plugins. If the plugin supports logs.                                                                                                                                                                                                                                                                                                                                                   |
 | `metrics`       | boolean                 | No       | For data source plugins. If the plugin supports metric queries. Used in the Explore feature.                                                                                                                                                                                                                                                                                                            |
-| `mixed`         | boolean                 | No       | Not to be used by external plugins. Special property for the built-in mixed plugin.                                                                                                                                                                                                                                                                                                                     |
```

Comment:
Any reason to remove `mixed` support ? we do have external plugin that uses `mixed` type https://github.com/GoshPosh/grafana-meta-queries

Reply:
cc/ @marcusolsson Any idea? 🤔